modules/ovn-kubernetes-logical-architecture-con.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: CONCEPT
+[id="ovn-kubernetes-logical-architecture-con_{context}"]
+= OVN-Kubernetes logical architecture
+
+The following diagram shows the OVN-Kubernetes logical architecture.
```

Comment:
Without a more comprehensive description of the architecture that this module is about, it's not accessible. If I'm using a screen reader, the alt-text doesn't give me enough and the key components list doesn't describe how things function as a system.